app/models/user.rb

```diff
@@ -16,7 +16,9 @@ class User < ActiveRecord::Base
 
   validates_lengths_from_database :except => [:firstname, :lastname, :format, :mail, :login]
   attr_accessor :password, :password_confirmation, :current_password
+  attr_reader :password_changed
   after_save :ensure_default_role
+  after_save :unset_password_changed
```

Comment:
@dhlavac When I see your tests - it looks like you're calling at most `.valid?` and `.save`, so I'm not 100% positive this is ggcovered as per the comment above.